src/components/home/GoogleMapContainer.js

```diff
@@ -13,7 +13,7 @@ export const GoogleMapContainer = (props) => {
   let markers = props.targets;
   return (
     <div className="map">
-      <Map center={location} markers={markers} updateTargetInfo={props.actions.updateFreeTarget} newTarget={props.newTarget} topicsList={props.topics}/>
+      <Map center={location} markers={markers} updateTargetInfo={props.actions.updateFreeTarget} newTarget={props.newTarget} topicsList={props.topics} />
```

Comment:
Also, instead of repeat the word `props` you can declare each prop in the function declaration.
`export const GoogleMapContainer = ({ prop1, prop2, etc }) => {`